src/README.md

```diff
@@ -0,0 +1,9 @@
+# Javascript Session 1 starter files
+
+Starter files:
+- `index.html`
+- `index.js`
```

Comment:
Can you explain the benefit of this `index.js` to a student?

Reply:
Um, I think that’s a carry-over from my test repo that I imported. Will delete this file.